Package.swift

```diff
@@ -7,13 +7,12 @@ let package = Package(
        .macOS(.v10_15)
```

Comment:
I think we should update the swift version on line 1 (couldn't mark the row...)
Suggestion:
`// swift-tools-version:5.3`